build/dependencies.props

```diff
@@ -76,14 +76,14 @@
     <MicrosoftExtensionsWebEncodersPackageVersion>2.1.1</MicrosoftExtensionsWebEncodersPackageVersion>
 
     <!-- Packages that come from aspnet/Scaffolding. (Used only in the 2.1.x package archives.) -->
-    <MicrosoftVisualStudioWebCodeGenerationContractsPackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGenerationContractsPackageVersion>
-    <MicrosoftVisualStudioWebCodeGenerationCorePackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGenerationCorePackageVersion>
-    <MicrosoftVisualStudioWebCodeGenerationDesignPackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGenerationDesignPackageVersion>
-    <MicrosoftVisualStudioWebCodeGenerationEntityFrameworkCorePackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGenerationEntityFrameworkCorePackageVersion>
-    <MicrosoftVisualStudioWebCodeGenerationPackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGenerationPackageVersion>
-    <MicrosoftVisualStudioWebCodeGenerationTemplatingPackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGenerationTemplatingPackageVersion>
-    <MicrosoftVisualStudioWebCodeGenerationUtilsPackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGenerationUtilsPackageVersion>
-    <MicrosoftVisualStudioWebCodeGeneratorsMvcPackageVersion>2.1.7</MicrosoftVisualStudioWebCodeGeneratorsMvcPackageVersion>
+    <MicrosoftVisualStudioWebCodeGenerationContractsPackageVersion>2.1.10</MicrosoftVisualStudioWebCodeGenerationContractsPackageVersion>
```

Comment:
Checked around and couldn't find a downside to the moves here and below to the latest patch versions.